### PR TITLE
feat(nuxt): Add nuxt wizard

### PR DESCRIPTION
**Pull request by andreiborza**
Closes: #705

### CODE REVIEW

File: src/nuxt/sdk-setup.ts [hunk 1]
```diff
@@ -0,0 +1,209 @@
...
+    // so we fall back to showing how to configure the nuxt config
+    // manually.
+    if (e instanceof MagicastError) {
+      clack.log.warn(
+        `Automatic configuration of ${chalk.cyan(
+          path.basename(config),
+        )} failed, please add the following settings:`,
+      );
+      // eslint-disable-next-line no-console
+      console.log(`\n\n${getNuxtModuleFallbackTemplate(options)}\n\n`);
```
Comment (Lms24) [inline src/nuxt/sdk-setup.ts:93]: l (feel free to disregard if N/A): We have a diff-like helper function for displaying code snippets: `makeCodeSnippet` in `clack-utils.ts`. Might be useful here but I'll let you make the call if it's necessary.

File: src/nuxt/sdk-setup.ts [hunk 2]
```diff
@@ -0,0 +1,209 @@
...
+      }
+
+      if (shouldWriteFile) {
+        await fs.promises.writeFile(
+          path.join(process.cwd(), typeScriptDetected ? tsConfig : jsConfig),
+          getSentryConfigContents(dsn, configVariant, selectedFeatures),
+          { encoding: 'utf8', flag: 'w' },
+        );
+        clack.log.success(
+          `Created fresh ${chalk.cyan(
```
Comment (Lms24) [inline src/nuxt/sdk-setup.ts:189]: l/personal preference: 

```suggestion
          `Created new ${chalk.cyan(
```

**Review by Lms24 — APPROVED** [review]
I used the wizard in an OSS nuxt project and it worked well! The steps seem fine to me, wording is great! :shipit:

File: src/nuxt/sdk-setup.ts (hunk 1 above)
Comment (andreiborza) [reply src/nuxt/sdk-setup.ts:93]: Thanks, I'll disregard for now to get this out, but will revisit.

File: e2e-tests/test-applications/nuxt-4-test-app
Comment (onurtemizkan) [inline e2e-tests/test-applications/nuxt-4-test-app:1]: @andreiborza, this test app is added here as a submodule. I think that's why the Nuxt tests are broken on master.

Comment (andreiborza) [reply e2e-tests/test-applications/nuxt-4-test-app:1]: Thanks, nice catch @onurtemizkan, I opened up a pr to fix it: https://github.com/getsentry/sentry-wizard/pull/742

**Pull request merged**